projects/scannet_offline_eval/demo/components/app.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import logging
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict
+
+import cv2
+import dash
+import dash_bootstrap_components as dbc
+import numpy as np
+import torch
+
+# from dash_extensions.websockets import SocketPool, run_server
+from dash_extensions.enrich import BlockingCallbackTransform, DashProxy
+from loguru import logger
+from torch_geometric.nn.pool.voxel_grid import voxel_grid
+
+from home_robot.core.interfaces import Observations
+from home_robot.mapping.voxel.voxel import SparseVoxelMap
+from home_robot.utils.point_cloud_torch import get_bounds
+
+from .directory_watcher import DirectoryWatcher
+
+
+@dataclass
+class AppConfig:
+    pointcloud_update_freq_ms: int = 2000
+    video_feed_update_freq_ms: int = 300
+
+    directory_watcher_update_freq_ms: int = 300
+    directory_watch_path: str = "publishers/published_trajectory/obs"
+    pointcloud_voxel_size: float = 0.05
+
```

Comment:
I think a lot of these files make more sense in home_robot/datasets or something right?